RealmTasks Android/app/src/main/res/layout/item_row.xml

```diff
@@ -30,48 +30,79 @@
-            android:textColor="@android:color/white"
-            android:textSize="@dimen/row_text_size"/>
-
-        <EditText
-            android:id="@+id/edit_text"
+        <RelativeLayout
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_centerVertical="true"
-            android:layout_marginLeft="@dimen/row_text_margin_left"
-            android:layout_marginRight="@dimen/row_text_margin_right"
-            android:textColor="@android:color/white"
-            android:visibility="gone"/>
+            android:layout_height="wrap_content">
 
-        <TextView
-            android:id="@+id/badge"
-            android:layout_width="@dimen/row_badge_width"
-            android:layout_height="match_parent"
-            android:layout_alignParentRight="true"
-            android:background="#26FFFFFF"
-            android:gravity="center"
-            android:text="@string/badge_placeholder"
-            android:textColor="@android:color/white"
-            android:textSize="@dimen/row_text_size"
-            android:visibility="gone"/>
+            <TextView
+                android:id="@+id/text"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_alignParentLeft="true"
+                android:layout_centerVertical="true"
+                android:layout_marginLeft="@dimen/row_text_margin_left"
+                android:layout_marginRight="@dimen/row_text_margin_right"
+                android:paddingTop="@dimen/row_text_padding"
+                android:text="@string/row_text_placeholder"
+                android:textColor="@android:color/white"
+                android:textSize="@dimen/row_text_size"
+                tools:text="Go shopping at 5pm"
+                />
 
-    </RelativeLayout>
+            <EditText
+                android:id="@+id/edit_text"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_centerVertical="true"
+                android:layout_marginLeft="@dimen/row_text_margin_left"
+                android:layout_marginRight="@dimen/row_text_margin_right"
+                android:textColor="@android:color/white"
+                android:visibility="gone"/>
+
+            <TextView
+                android:id="@+id/badge"
+                android:layout_width="@dimen/row_badge_width"
+                android:layout_height="match_parent"
+                android:layout_alignParentRight="true"
+                android:background="#26FFFFFF"
+                android:gravity="center"
+                android:text="@string/badge_placeholder"
+                android:textColor="@android:color/white"
+                android:textSize="@dimen/row_text_size"
+                android:visibility="gone"
+                />
+
+        </RelativeLayout>
+
+        <RelativeLayout
```

Comment:
I debated with myself on this.  Given only one child view (the TextView) it's not necessary right now, but was thinking if it grew to include additional UI elements, it would be good to have this "metadata" information in a container row.  I"ll change it to a FrameLayout though.